default/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 2.8.12)
 project(cmake_wrapper)
 
-include(conanbuildinfo.cmake)
+include("${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
```

Comment:
> I use the package development flow when developing the recipes.
When building, this does not copy the sources from self.source_folder to self.build_folder,
so the include requires an explicit path.

Which work flow to you mean exactly? @madebr

Reply:
https://docs.conan.io/en/latest/developing_packages/package_dev_flow.html

I create a `src`, `bin` and `pkg` folder, and run `conan source -sf src`, `conan install -if bin $PATH_TO_CONAN madebr/testing`, `conan build -if bin -sf src -bf bin -pf pkg` and `conan package -if bin -sf src -bf bin -pf pkg` while developing the recipe.

Because I can easily remove/change source, build and package folders, cmake must know the correct path.

Reply:
(This is also why caching downloads would be useful)

Reply:
I still don't understand this to be honest. Conan is already creating `build`, `source`, `export` directories etc. Why can't you just use them?

Reply:
I think @madebr uses a different approach. For `conan create` it's fine, but when he needs to execute all steps the conanfileinfo.cmake won't work, that's why he changed. is it correct @madebr ?

Reply:
Indeed @uilianries , When using the [development flow](https://docs.conan.io/en/latest/developing_packages/package_dev_flow.html), you can specify the source, binary and package folders. I use it to quickly iterate on the source, build, package ... functions.
These folders are not created automatically in this method.

Reply:
> These folders are not created automatically in this method.

Thanks, I wasn't aware of this part 😄